_includes/misc/support-powerpaste.md

```diff
@@ -1,30 +1,30 @@
-## PowerPaste support (Word Copy and Paste)
+## PowerPaste support (Copy and Paste)
 
-{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from versions of Microsoft Word and Microsoft Excel:
+{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from Google Docs and versions of Microsoft Word and Microsoft Excel:
 
 * Covered by Microsoft Mainstream Support based on the [Microsoft Fixed Lifecycle Policy](https://support.microsoft.com/en-us/help/14085/fixed-lifecycle-policy).
 * Provided by Microsoft Office 365.
 
 ### PowerPaste Feature and Browser Support
 
-PowerPaste (Word copy and paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There is some variances of functionality for different browsers.
+PowerPaste (Copy and Paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There is some variances of functionality for different browsers.
 
 |                       | HTML Cleaning | Improved HTML Cleaning | Image Import  |
 |-----------------------| :-----------: | :--------------------: | :-----------: |
 | Internet Explorer 11  | {{site.tick}} | {{site.cross}}         | {{site.cross}}|
 | Microsoft Edge Latest | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
 | Chrome Latest         | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
 | Firefox Latest        | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
-| Safari Latest         | {{site.tick}} | {{site.tick}}          | {{site.cross}}|
+| Safari Latest         | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
```

Comment:
@metricjs as far as I know this has been working for quite a while now, so I've changed it but just wanted to confirm?

Reply:
Unless "Image Import" means something different to what I'm thinking of, this works, yes.